src/Polly.Extensions/DependencyInjection/AddResiliencePipelineContext.cs

```diff
@@ -61,8 +61,7 @@ internal AddResiliencePipelineContext(ConfigureBuilderContext<TKey> registryCont
     public TOptions GetOptions<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicParameterlessConstructor)] TOptions>(string? name = null)
     {
         var monitor = ServiceProvider.GetRequiredService<IOptionsMonitor<TOptions>>();
-
-        return name == null ? monitor.CurrentValue : monitor.Get(name);
+        return monitor.Get(name);
```

Comment:
Effectively duplicates [the implementation](https://github.com/dotnet/runtime/blob/24e7d1b0959234a36fb09a5c64cfd2da20f10618/src/libraries/Microsoft.Extensions.Options/src/OptionsMonitor.cs#L72-L102) the way it was.